app/javascript/chat/chat.jsx

```diff
@@ -38,6 +39,11 @@ export default class Chat extends Component {
     const chatChannels = JSON.parse(props.chatChannels);
     const chatOptions = JSON.parse(props.chatOptions);
 
+    this.debouncedChannelFilter = debounce(
+      this.triggerChannelFilter.bind(this),
+      300,
```

Comment:
Lowered to 300 ms to make it a bit snappier but I like waiting until the user is done typing before doing the lookup otherwise it feels like its interrupting as you type

Reply:
We use 300ms in the reading list as well, +1 :)